webidl/MathMLLinkableElement.idl

```diff
@@ -0,0 +1,16 @@
+[Exposed=Window]
+interface MathMLLinkableElement : MathMLElement { 
+  [CEReactions] attribute DOMString target;
```

Comment:
It seems like this should closely follow:
https://html.spec.whatwg.org/#the-link-element.
So then href may be better?

Reply:
based on https://html.spec.whatwg.org/#the-a-element, not link